dandi/dandiapi.py

```diff
@@ -1154,6 +1177,8 @@ def get_assets_with_path_prefix(
         Returns an iterator of all assets in this version of the Dandiset whose
         `~RemoteAsset.path` attributes start with ``path``
 
+        ``path`` is normalized first to possibly remove leading `./` or relative
+        paths (e.g., `../`) within it.
```

Comment:
Add a blank line after this line to force a paragraph break.

Also, this paragraph should probably also be added to the docs for `get_asset_by_path()` and `download_directory()`, since those pass their path arguments to this method.

Reply:
ok, will add use to `get_asset_by_path` as well